vars/applyTemplate.groovy

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env groovy
+
+class ApplyTemplateInput implements Serializable{
+  String projectName
+  String templateFile
+  String parameterFile
```

Comment:
It seems that a default value of `""` for the `parameterFile` will let you apply a template without a parameter file. Command line test with `--param-file "" --ignore-unknown-parameters` will let me use the default value for a parameter specified in the template.